docs/source/get_involved/governance.rst

```diff
@@ -740,7 +740,7 @@ References
 Our governance model is inspired by various existing governance
 structures. In particular, we'd like to acknowledge:
 
-* scikit-learn's `governance model <https://www.sktime.net/en/latest/governance.html>`__
+* scikit-learn's `governance model <https://scikit-learn.org/stable/governance.html>`__
```

Comment:
Not sure why `sktime` link was used here. Changed to that of `scikit-learn`, please confirm.

Reply:
confirmed, yes, of course